synapse/storage/state.py

```diff
@@ -189,19 +190,17 @@ def _get_filtered_current_state_ids_txn(txn):
 
                 if include_other_types:
                     unique_types = set(filtered_types)
-                    clause_to_args.append(
-                        (
-                            "AND type <> ? " * len(unique_types),
-                            list(unique_types)
-                        )
-                    )
+                    sql += " AND type <> ? " * len(unique_types)
+                    additional_args = list(unique_types)
```

Comment:
it's not obvious to me that this needs copying: why not just `additional_args = unique_types` ?